app/src/main/res/values/strings.xml

```diff
@@ -124,17 +123,17 @@
 
     <!-- activity description -->
     <string name="activity_description_title">Title</string>
-    <string name="activity_description_by">By </string>
-    <string name="activity_description_with"> with </string>
-    <string name="activity_description_during"> during </string>
-    <string name="activity_description_hours"> hours </string>
+    <string name="activity_description_by">By &#160;</string>
```

Comment:
What are those signs such as "&#160"? Are they equivalent to a space?

Reply:
Yes it is a space but coded like that, it is taken into account during the display whereas a simple space at the end of the string is not taken into account.